spec/services/submit_to_app_store/prior_authority_payload_builder_spec.rb

```diff
@@ -53,8 +53,8 @@
         },
         solicitor: {
           contact_email: 'james@email.com',
-          contact_full_name: 'James Blake',
-          full_name: 'Richard Jenkins',
```

Comment:
we have dropped the solicitor name here

Reply:
As above - we only collect one solicitor name in PA, which we call the contact name. The solicitor model has 2 name fields, one prefixed with 'contact'. There's no point sending the other one over, as it will always be nil